Include Post data on the response object when declared  #2416

websupporter opened the pull request:
See https://github.com/WP-API/WP-API/issues/2416

websupporter committed:
Only include Post data on the response object when declared in schema #2416

danielbachhuber commented:
Sorry, this isn't quite the right approach. We should permit arbitrary data to be included in the response — however the core fields shouldn't be included by default unless they're a part of the schema. See how the `title` field is included as an example of how the others should follow.

websupporter commented:
thanks @danielbachhuber, will rework my approach.

websupporter committed:
Include Post data on the response object when declared #2416

the core fields shouldn't be included by default unless they're a part of the schema

danielbachhuber commented:
@websupporter Sorry, I missed your updates. Feel free to ping me next time so the PR isn't left hanging.
